src/cmd_all/src/standalone.rs

```diff
@@ -215,7 +216,11 @@ pub async fn standalone(
     }
     if let Some(opts) = compute_opts {
         tracing::info!("starting compute-node thread with cli args: {:?}", opts);
-        let _compute_handle = tokio::spawn(async move { risingwave_compute::start(opts).await });
+        // TODO(shutdown): pass the shutdown token
```

Comment:
By this I suppose you mean that there will be one shutdown token instantiated for the entire process, and it should be passed to each of the embedded nodes?

Reply:
Yes, exactly.